lib/segment/src/common/anonymize.rs

```diff
@@ -67,6 +67,8 @@ impl Anonymize for DateTime<Utc> {
     fn anonymize(&self) -> Self {
         let coeff: f32 = rand::random();
 
-        *self + chrono::Duration::seconds(((coeff * 20.0) - 10.0) as i64)
+        *self
+            + chrono::Duration::try_seconds(((coeff * 20.0) - 10.0) as i64)
+                .expect("Failed to create TimeDelta")
```

Comment:
Shall we do `unwrap_or_default` to prevent hard panics if this ever fails? With this, if it would fail, it would leave the time unchanged.

```suggestion
        *self + chrono::Duration::try_seconds(((coeff * 20.0) - 10.0) as i64).unwrap_or_default()
```